api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -42,7 +42,16 @@
 	
 	private static final Logger log = LoggerFactory.getLogger(OpenmrsConstants.class);
 	
-	public static String KEY_OPENMRS_APPLICATION_DATA_DIRECTORY = "OPENMRS_APPLICATION_DATA_DIRECTORY";
+	
+	private static String KEY_OPENMRS_APPLICATION_DATA_DIRECTORY = "OPENMRS_APPLICATION_DATA_DIRECTORY";
+	
+	public static void SET_KEY_OPENMRS_APPLICATION_DATA_DIRECTORY(String name){
+		KEY_OPENMRS_APPLICATION_DATA_DIRECTORY = name;
```

Comment:
The change was not specifically stated in the ticket. But by changing KEY_OPENMRS_APPLICATION_DATA_DIRECTORY to a public static final blocked access to other parts of the code, resulting in a failed compile. So I changed it to private static and added the get and set methods. I will take a look at the conventions to see what I need to fix.